packages/rspeedy/plugin-react-alias/src/index.ts

```diff
@@ -35,11 +36,17 @@ export function pluginReactAlias(options: Options): RsbuildPlugin {
 
       const require = createRequire(import.meta.url)
 
-      const reactLynxDir = path.dirname(
-        require.resolve('@lynx-js/react/package.json', {
-          paths: [rootPath ?? api.context.rootPath],
-        }),
-      )
+      const reactLynxPkg = require.resolve('@lynx-js/react/package.json', {
+        paths: [rootPath ?? api.context.rootPath],
+      })
+      const reactLynxPkgContent = require(reactLynxPkg) as { version?: string }
+      const version = reactLynxPkgContent?.version
+
+      if (!version) {
+        throw new Error('version field not found in @lynx-js/react package')
+      }
```

Comment:
Do we really have `@lynx-js/react/package.json` without `version`? I would prefer let it crash instead of try-catch everywhere.